src/Views/Admin/Pages/Reports.php

```diff
@@ -28,9 +28,10 @@ public function init()
 
     public function __construct()
     {
-        // Do nothing
+
     }
```

Comment:
Then... why does this exist?

Reply:
Resolved: https://github.com/impress-org/givewp/pull/6796/commits/1ee1add569928cc953cddffd202be4ddff31a877